WORKSPACE

```diff
@@ -32,11 +32,15 @@ bind(
     actual = "//external:ssl",
 )
 
+# envoy commit date 04/10/2019
+# bazel version: 0.22.0
```

Comment:
nice touch

Reply:
It might be another pain if we forgot to update the comment... We will see.